script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -226,47 +241,51 @@ I want the other half――my half――to remain genuine until the very end.
 [sha:0db9e2248c8ee50a0a4c76a7469c35520e231126]{
 -- Page 399, Offset 34428.
 -- ……自分の命と引き換えにするにしては、滑稽な望みだと分かっていても。
-...I understand that giving up my own life in exchange for that wish is foolish.
+I understand that this foolish wish will cost me my life...
+ but even that is a price I'll gladly pay.
 }
 [sha:fbca2377cb3ec797924f82b412490fdd2404a1d0]{
 -- Page 400, Offset 34429.
 -- 「―――そんなコト。そんなコトのために、命を投げ捨てるんですか。望みは。貴方の望みというのは、その程度のものなんですか」
-"―――That's it? You're telling me you're going to throw away your life away for something like that? Is your desire truly... that meager?"
+"―――That's it?
+ You're really going to throw your life away over something like that?
+ Is that all...?
+ Is that really all you wish for?"
 }
 [sha:d2ddc5b03c30d69f749013f63690ce5d0dc5397f]{
 -- Page 400, Offset 34430.
 -- 「……そっか。やっぱり小さい、かな」
-"...You're right. I guess I was thinking a little small."
+"You're right... Maybe I should have dreamed bigger."
```

Comment:
Flows fine to me? Suggestion also doesn't feel very Shiki-rashii, at least to me
> C: Is that really all you wish for?
> S: You're right... Maybe I should have dreamed bigger.

Reply:
I struggled to come up with a word for 小さい that works for both of the cases, since he echoes this line later. That's probably why it doesn't sound very "Shiki". That said, the flow is a bit weird.

"You're right" isn't a direct answer to her question of "Is that all you wish for?", but of course, he doesn't actually provide one here. He's just like "You would think it is a insignificant thing to wish for, huh?", more as commentary to her line. I guess that's more why I felt like "That's right" doesn't quite match. "dream bigger" works here, but I couldn't make it work later when it's echoed. Well...

Reply:
I mean yeah, it's a slant rhyme, but it's a pretty normal conversation pattern I feel. As for the linkage, you could do
> ...maybe I should have dreamed bigger.
> ...
> The thing I wished for more than anything―――
> It wasn't a big dream at all. All I wanted...

or maybe
> You're right... I guess I wasn't asking for much, huh.
> ...
> The thing I wished for more than anything―――
> I wasn't asking for much. All I wanted

Reply:
Well, the first option certainly is better.